src/ontology/cl-edit.owl

```diff
@@ -32490,6 +32491,16 @@ AnnotationAssertion(terms:date obo:CL_4047012 "2024-09-06T08:41:39Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047012 "angiogenic pericyte"@en)
 SubClassOf(obo:CL_4047012 obo:CL_0000669)
 
+# Class: obo:CL_4047023 (lamina propria fibroblast)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36251739") Annotation(oboInOwl:hasDbXref "PMID:36639348") obo:IAO_0000115 obo:CL_4047023 "A fibroblast located in the lamina propria of the intestinal mucosa, particularly in the small intestine. This cell expresses PDGFRα and CD81 and is negative for α-smooth muscle actin (α-SMA). This cell is positioned adjacent to myofibroblasts surrounding colonic crypts. It is capable of synthesizing extracellular matrix components and structural proteins such as collagen and elastin.")
```

Comment:
Looks like PMID:36251739 and PMID:36639348 are typos (not papers about fibroblasts)

"particularly in the small intestine" and "positioned adjacent to myofibroblasts surrounding colonic crypts" seem to contradict each other.  Are they perhaps more numerous in the SI?  Are the ones in the colon all or mostly localised to myofibroblsts surrounding crypts?